### PR TITLE
Fix JS name and remove references to .map files

### PR DESCRIPTION
**Pull request by szepeviktor**
While doing typo fixes I've discovered `--exclude=static/js/*.min.js` misses zxcvbn.

Watch my next PR! 👁️

### CODE REVIEW

File: templates/accounts/set_password.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h1>Set a Password</h1>
 
 {% block scripts %}
 {% compress js %}
-<script src="{% static 'js/zxcvbn.js' %}"></script>
+<script src="{% static 'js/zxcvbn.min.js' %}"></script>
 <script src="{% static 'js/set-password.js' %}"></script>
 {% endcompress %}
```
Comment (szepeviktor) [inline templates/accounts/set_password.html:49]: Should I move zxcvbn out of `compress js` ??

Comment (cuu508) [reply templates/accounts/set_password.html:49]: No, `{% compress %}` bundles `zxcvbn` and `set-password` into a single file, which is what we want.

Comment (szepeviktor) [reply templates/accounts/set_password.html:49]: Done 🙃

**Comment by cuu508** [comment]
Thanks!

**Pull request merged**